_tuning-your-cluster/cluster.md

```diff
@@ -1,7 +1,9 @@
 ---
 layout: default
 title: Cluster formation
```

Comment:
Are we going with 'cluster formation' or should we rename to "Creating a cluster"?